doc/development/deprecations.rst

```diff
@@ -62,22 +55,29 @@ Pending deprecations
   `discussion forum <https://discuss.pennylane.ai/>`_.
 
   - Deprecated in v0.31
-  
-* The ``prep`` keyword argument in ``QuantumScript`` is deprecated and will be removed from ``QuantumScript``.
-  ``StatePrepBase`` operations should be placed at the beginning of the `ops` list instead.
+  - Changed in v0.33
 
-  - Deprecated in v0.33
-  - Will be removed in v0.34
+* The behaviour of ``Operator.__eq__`` and ``Operator.__hash__`` will be updated soon. Their documentation
+  has been updated to reflect the incoming changes.
```

Comment:
The old behaviour has been removed. 0.33 has `==` behaving the same as `qml.equal`.